packages/astro/src/@types/astro.ts

```diff
@@ -1317,7 +1317,7 @@ export interface AstroUserConfig {
 		 * {
 		 *   markdown: {
 		 *     // Example: Translate the footnotes text to another language, here are the default English values
-		 *     remarkRehype: { footnoteLabel: "Footnotes", footnoteBackLabel: "Back to content"},
+		 *     remarkRehype: { footnoteLabel: "Footnotes", footnoteBackLabel: "Back to reference 1"},
```

Comment:
This is a breaking change from https://github.com/syntax-tree/mdast-util-to-hast/commit/56c88e45690be138fad9f0bf367b939d09816863, but it's very unlikely to affect someone.

Reply:
Do I understand correctly that the breaking change is the default wording is now different?

This change in default looks good to me (more accessible than many identical links), but we should maybe include a link in the changeset, just in case someone wants to update their text. (I think it's likely to affect people, but unlikely that that affect is bad/undesired.)

Reply:
Yeah it's the changed default. I can add a changeset here, maybe together with that we've upgraded the remark/rehype plugins